src/api/runCucumber.ts

```diff
@@ -14,16 +14,19 @@ import { resolvePaths } from './paths'
 import { makeRuntime } from './runtime'
 import { initializeFormatters } from './formatters'
 import { getSupportCodeLibrary } from './support'
+import { Console } from 'console'
 
 export async function runCucumber(
   configuration: IRunConfiguration,
   {
     cwd = process.cwd(),
     stdout = process.stdout,
+    stderr = process.stderr,
     env = process.env,
   }: Partial<IRunEnvironment>,
   onMessage?: (message: Envelope) => void
 ): Promise<IRunResult> {
+  const logger = new Console(stdout, stderr)
```

Comment:
Not a blocker, but interested in thoughts on just doing `new Console(stderr)` so we can use whatever feels right of `log`/`info`/`warn`/`error` and it always goes to stderr?

Reply:
I'm not totally sure either, but it feels weird that e.g. we use `error` when outputting the published report URL. I'll leave this as it is for now as we're only using `warn` and `error` anyway, but maybe one to revisit.